src/features/wallet/Wallet.tsx

```diff
@@ -269,12 +271,26 @@ const Wallet: FC<Props> = ({ signingKeys }) => {
                 <span
                   onClick={() => {
                     if (votingContract && votesRevealed && multicallContract) {
-                      collectRewards(
+                      const unclaimedRewards = votesRevealed.filter(
```

Comment:
Does this fix the multicall gas issue?

Reply:
I think before it wasn't filtering between what you had claimed and what was revealed. 

So now say you have 40 claimed events, but 42 reveals -- obviously those 2 reveals are the eligible claimed rewards to try to claim